transport-msg.tex

```diff
@@ -683,16 +683,17 @@ \subsubsection{Device Reset and Shutdown}
 The driver \emph{MAY} reset or shut down a device at any time by writing a status
 of 0 via \msgref{SET_DEVICE_STATUS}. This forces the device to discard its state
 and any pending operations on virtqueues. Once a device is reset, the driver
-\emph{MUST} reinitialize it (see
+\emph{MAY} reinitialize the device (see
 \ref{sec:Virtio Transport Options / Virtio Over Messages / Device Initialization})
-before resuming normal operation.
+if it wishes to try and use the device again.
```

Comment:
not sure if the sentence is correct.
`if it wishes to try to use the device again.`?

Reply:
Replaced: "if it wishes to try to use the device again"
With: "if it wishes to use the device again"
in two places.

I am open to further wording improvements if anyone has them